app/controllers/experiments.js

```diff
@@ -41,32 +43,33 @@ export default Ember.Controller.extend({
         set (_, value) {
             var sort = this.get('sort');
             if (sort) {
-                var [prop, ] = sort.split(':');
-                this.set('sort', `${prop}:${value}`);
+                var prop = sort.replace(DESC, '');
+                this.set('sort', `${value}${prop}`);
             }
             else {
-                this.set('sort', `title:${value}`);
+                this.set('sort', `${value}title`);
             }
+            return value;
         }
     }),
     toggleOrder: function(order) {
-        if (order === 'asc') {
-            this.set('sortOrder', 'desc');
+        if (order === ASC) {
+            this.set('sortOrder', DESC);
         } else {
-            this.set('sortOrder', 'asc');
+            this.set('sortOrder', ASC);
         }
     },
     actions: {
         selectStatusFilter: function(status) {
             this.set('active', status);
             this.set('sortProperty', 'title');
-            this.set('sortOrder', 'asc');
+            this.set('sortOrder', ASC);
```

Comment:
May not be in the scope of this PR but I'm pretty sure the _search endpoint doesn't have a filter parameter so the status filter may need to be adjusted.

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-uri-request.html